airbyte-cdk/python/airbyte_cdk/sources/concurrent_source/thread_pool_manager.py

```diff
@@ -64,7 +64,11 @@ def _prune_futures(self, futures: List[Future[Any]]) -> None:
                     if optional_exception:
                         # Exception handling should be done in the main thread. Hence, we only store the exception and expect the main
                         # thread to call raise_if_exception
-                        self._most_recently_seen_exception = RuntimeError(f"Failed reading with error: {optional_exception}")
+                        # We do not expect this error to happen. The futures created during concurrent syncs should catch the exception and
+                        # push it to the queue. If this exception occurs, please review the futures and how they handle exceptions.
+                        self._most_recently_seen_exception = RuntimeError(
+                            f"Failed processing a future: {optional_exception}. Please contact the Airbyte team."
```

Comment:
Let's clarify how the users should contact our team. If this situation is a bug and/or requires work from the CDK team, we should ask folks to file an issue on airbytehq/airbyte, right?

Reply:
I can't confirm as I'm not sure what is our official support process. Maybe @girarda can chime in?